Client/Frontend/Browser/BluetoothKeyboardManager.swift

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+struct BluetoothKeyboardNotification {
+    static let TabTrayDidOpen = "TabTrayDidOpen"
+}
+
+@objc protocol BluetoothKeyboardDelegate : class {
+    optional func openNewTab()
+    optional func testFunc()
+}
+
+class WeakBluetoothKeyboardDelegate {
+    weak var value : BluetoothKeyboardDelegate?
+
+    init (value: BluetoothKeyboardDelegate) {
+        self.value = value
+    }
+
+    func get() -> BluetoothKeyboardDelegate? {
+        return value
+    }
+}
+
+class BluetoothKeyboardManager: NSObject {
```

Comment:
I'm not sure we need a manager of any kind to handle the keyboard events. Keyboard shortcuts are context-sensitive, so I think it would make more sense for each `UIViewController` to be responsible for its own set of shortcuts rather than delegating to a global manager (i.e., Command+T might mean something different in another view controller). So to simplify this PR, can we drop the `BluetoothKeyboardManager` altogether, then simply attach the view-specific `keyCommands` where they're needed?